.php_cs.dist

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()->in(__DIR__);
+
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PSR2' => true,
```

Comment:
We could of course use `@Symfony` here, if we wanted, but using `@PSR2` is probably a good start.

Reply:
I think using PSR2 should be enough.